### ISSUE TITLE
Feature: make sandwhiches stackable

### ISSUE THREAD

**Issue by milkev** [FoundationGames/Sandwichable#51]
hi! i saw some bug issues mildly related to this that have been closed. 
but is it possible to make sandwiches of the same components stack? the sandwiches are awesome and a great food source, but even just a stack of anything else will be better because they take up so much space.

**Comment by FoundationGames** [comment]
They were in 1.0.0-1.0.1.
It was removed due to a very strange dupe bug. I changed a lot of code in that area so I doubt that duping would be an issue anymore, but I don't really feel like sandwiches should be stackable. I could add it as a config option, however.
And please remember that sandwiches are going to be extremely buffed in the next update, with the inclusion of Meal API allowing it to fill beyond your normal hunger and saturation.

**Comment by milkev** [comment]
ooo, if it was like that i wouldnt mind stacking so much, especially with the sandwiches just... almost garaunteed to fill your entire good and saturation.
do add it as a config if its not to much trouble though! that would still be nice to have as an option.

**Comment by copygirl** [comment]
It would be nice to have as a config option. Perhaps you could try to make sandwiches less "overpowered" (especially when stackable) by limiting the amount of same / similar ingredients and maximum amount of items in a sandwich - those options could also be configurable.

**Comment by Snoe0** [comment]
Yes! A config option would be amazing.

**Comment by tankyap** [comment]
If you had a overpowered sandwiches on your inventory, this would be more overpowered but still add the config would be good.

**Comment by fzzyhmstrs** [comment]
Yes please bring this back, at least as a config option. I'm trying to make a "balanced meal" for Nourish that hits all five categories, but it's not worth it if the food doesn't stack. The balanced sandwich I'm trying to make isn't even that "OP". Something like 6 hunger and 5.5 saturation.
